### ISSUE TITLE
Can i get the other column value in column formatter function

### ISSUE THREAD

**Issue by faisalshah1** [fooplugins/FooTable#658]
Hello, Hope you are doing well.

I have two columns, Column A and Column B. and i have set the formatter in Column B so i get the value of Column B in that formatter function but can i get the value of Column A as well for that particular row.

**Comment by poulnike** [comment]
#656

**Comment by steveush** [comment]
This is now resolved with the release of 3.1.5. As per #656 the base column formatter function now provides the `rowData` object as the third parameter.

```javascript
function formatter(value, options, rowData){
	return ...;
}
```

Thanks
